zirgen/circuit/verify/wrap_zirgen.cpp

```diff
@@ -231,18 +245,17 @@ Val CircuitInterfaceZirgen::compute_poly(llvm::ArrayRef<Val> u,
   return Val(result);
 }
 
-std::unique_ptr<CircuitInterface> getInterfaceZirgen(mlir::ModuleOp mod,
-                                                     ProtocolInfo protocolInfo) {
-  return std::make_unique<CircuitInterfaceZirgen>(mod, protocolInfo);
+std::unique_ptr<CircuitInterface> getInterfaceZirgen(mlir::ModuleOp mod) {
+  return std::make_unique<CircuitInterfaceZirgen>(mod);
 }
 
-std::unique_ptr<CircuitInterface>
-getInterfaceZirgen(mlir::MLIRContext* ctx, mlir::StringRef filename, ProtocolInfo protocolInfo) {
+std::unique_ptr<CircuitInterface> getInterfaceZirgen(mlir::MLIRContext* ctx,
+                                                     mlir::StringRef filename) {
   ParserConfig config(ctx);
   OwningOpRef<ModuleOp> mod = parseSourceFile<ModuleOp>(filename, config);
   if (!mod)
     return nullptr;
-  return std::make_unique<CircuitInterfaceZirgen>(std::move(mod), protocolInfo);
+  return std::make_unique<CircuitInterfaceZirgen>(std::move(mod));
```

Comment:
I'm rather confused about what's going on here, could you describe what the purpose of this file is?

Reply:
Added comments